IbisPlugins/CameraCalibration/cameracalibrationsidepanelwidget.cpp

```diff
@@ -107,7 +107,7 @@ void CameraCalibrationSidePanelWidget::UpdateUi()
     ui->viewListWidget->blockSignals( true );
     ui->viewListWidget->clear();
     CameraCalibrator * calib = m_pluginInterface->GetCameraCalibrator();
-    for( int i = 0; i < calib->GetNumberOfViews(); ++i )
+    for( int i = 0; i < calib->GetNumberOfAccumulatedViews(); ++i )
```

Comment:
I don't think this change is right. The accumulation mechanism is used to concatenate multiple camera matrices and images that we average together to produce a single view. Here, we want to list the views.